notebooks/02-ingestion/12-spatial-dimensions.ipynb

```diff
@@ -0,0 +1,711 @@
+    "  }\n",
+    "```\n",
+    "\n",
+    "Run the following cell to test a polygon filter:\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "618bbf5e-0cf3-4ad9-8a04-0452098ab042",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# this polygon filter is roughly the outline of Manhattan\n",
+    "polygon_filter_query = {\n",
+    "  \"queryType\": \"topN\",\n",
+    "  \"dataSource\": {\n",
+    "    \"type\": \"table\",\n",
+    "    \"name\": \"example-spatial-index\"\n",
+    "  },\n",
+    "  \"dimension\": {\n",
+    "    \"type\": \"default\",\n",
+    "    \"dimension\": \"marital_status\",\n",
+    "    \"outputName\": \"marital_status\",\n",
+    "    \"outputType\": \"STRING\"\n",
+    "  },\n",
+    "  \"metric\": {\n",
+    "    \"type\": \"dimension\",\n",
+    "    \"ordering\": {\n",
+    "      \"type\": \"lexicographic\"\n",
+    "    }\n",
+    "  },\n",
+    "  \"threshold\": 1001,\n",
+    "  \"intervals\": {\n",
+    "    \"type\": \"intervals\",\n",
+    "    \"intervals\": [\n",
+    "      \"-146136543-09-08T08:23:32.096Z/146140482-04-24T15:36:27.903Z\"\n",
+    "    ]\n",
+    "  },\n",
+    "  \"filter\": {\n",
+    "    \"type\": \"spatial\",\n",
+    "    \"dimension\": \"address_spatial\",\n",
+    "    \"bound\": {\n",
+    "        \"type\": \"polygon\",\n",
+    "        \"abscissa\": [30.942493, 36.917158, 22.799200, 3.914715, -0.286389, -35.494284, 11.654848, 30.942493 ],\n",
+    "        \"ordinate\": [32.434730, 9.758948 ,-16.959803,-8.524121,  7.766978,  19.844799, 51.309644, 32.434730 ]\n",
+    "      }\n",
+    "  },\n",
+    "  \"granularity\": {\n",
+    "    \"type\": \"all\"\n",
+    "  },\n",
+    "  \"aggregations\": [\n",
+    "    {\n",
+    "      \"type\": \"doubleMin\",\n",
+    "      \"name\": \"min_income\",\n",
+    "      \"fieldName\": \"income\"\n",
+    "    },\n",
+    "    {\n",
+    "      \"type\": \"doubleMax\",\n",
+    "      \"name\": \"max_income\",\n",
+    "      \"fieldName\": \"income\"\n",
+    "    },\n",
+    "    {\n",
+    "      \"type\": \"count\",\n",
+    "      \"name\": \"user_count\"\n",
+    "    }\n",
+    "  ],\n",
+    "  \"context\": {\n",
+    "    \"queryId\": \"2dcbd3c7-cebe-4bb9-8b8b-d1887deda9ea\",\n",
+    "    \"sqlOuterLimit\": 1001,\n",
+    "    \"sqlQueryId\": \"2dcbd3c7-cebe-4bb9-8b8b-d1887deda9ea\",\n",
+    "    \"useNativeQueryExplain\": False\n",
+    "  }\n",
+    "}\n",
+    "\n",
+    "headers = {\n",
+    "  'Content-Type': 'application/json'\n",
+    "}\n",
+    "\n",
+    "result = rest_client.post(\"/druid/v2\", json.dumps(polygon_filter_query), headers=headers)\n",
+    "\n",
+    "json.loads(result.text)[0]['result']\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "98846a26-f46d-47ef-ad4c-e09a096367c6",
+   "metadata": {},
+   "source": [
+    "## Cleanup \n",
+    "The following cell stops data generation, ingestion jobs and removes the datasource from Druid."
```

Comment:
```suggestion
    "The following cell stops data generation, ingestion jobs, and removes the datasource from Druid."
```

Reply:
changed for "The following cell removes the example datasource from Druid."